templates/cdk.master.auth-webhook.py

```diff
@@ -145,7 +145,7 @@ def check_secrets(token_review):
                 username = uid = b64decode(username_b64).decode('UTF-8')
                 pw_delim = '::'
                 if pw_delim in password:
-                    uid = password.split(pw_delim)[0]
+                    uid = password.rsplit(pw_delim, 1)[0]
```

Comment:
Do we need this delimiter splitting at all? It looks like uid is already stored as a separate field in the secret. Why don't we get it from there?

Reply:
> Do we need this delimiter splitting at all? It looks like uid is already stored as a separate field in the secret. Why don't we get it from there?

I like it as a separator in kubeconfig files, so i see:
```
token: worker-a/3::r0DzvhNmcWqDLtP72l8Ux2ZJNm5DvKJp
```
instead of:
```
token: worker-a/3r0DzvhNmcWqDLtP72l8Ux2ZJNm5DvKJp
```